src/System.Private.CoreLib/shared/System/TimeZoneInfo.Unix.cs

```diff
@@ -413,24 +415,34 @@ private static IEnumerable<string> EnumerateFilesRecursively(string path)
         {
             List<string> toExplore = null; // List used as a stack
 
+            int bufferSize = Interop.Sys.GetReadDirRBufferSize();
             string currentPath = path;
             for(;;)
             {
```

Comment:
`SafeDirectoryHandle` is no longer used anywhere after this change. You can delete it.